documentation/docs/contribute.md

```diff
@@ -8,41 +8,48 @@ keywords:
 - contribute
 - discord
 - GitHub
+- reference
 ---
 # Contribute to the Project
 
-**Thanks for thinking about contributing to the project! We have the following ways that you can contribute.**
+**Thank you for thinking about contributing to the project! You can contribute in the following ways:**
```

Comment:
The first sentence doesn't roll smoothly. "Thank you for thinking" is a bit of a tongue twister and all and all it is a long chain-like structure.

Reply:
Can't say that I agree, but we can always rephrase it :)   @charlesthompson3, what do you think? 

btw, I think you mean all in all.

Reply:
It has a lot of hard consonants, I'll give it that. But otherwise, not a reworkable offense. Maybe swap out one of the contributes for something else but if it takes more than six seconds to think about, I'd stick with new update.

Reply:
I'm drawing a blank, but if anyone has any suggestions please add them

Reply:
Hmm. 

> Thank you for considering a contribution to this project. You can help it in the following ways:

I am not sure if the change is worth it, but at least it gets rid of -ing -ing.